composer.json

```diff
@@ -57,3 +86,16 @@
         "sort-packages": true
     }
 }
+
+{
```

Comment:
This is causing the CI to fail due to the JSON being invalid (missing `,` after the above closing `}`).